README.md

```diff
@@ -16,3 +16,25 @@ As a user, I would like to configure my feature to use a single global content s
 
 ## What's next?
 Now that we have a feature that renders based on the global content instead of being fixed to component level content, we want to see how we can dynamically route content using PageBuilder templates. Switch to branch `fusion-12` to learn more about this next step.
+
+## New Task: User Story 12
+As a user, I would like to be able to render many pieces of content via this template in the browser.
+
+## GUIDE
+No code changes are required to complete this task
+
+1. Create a new template in PB called `Articles`
+
+2. Using the basic layout, add in the header feature and any other feature you'd like
+
+3. Under "Developer Tools" > "Resolvers", create a new resolver
```

Comment:
Linking to the docs on resolvers could be helpful to provide more context: https://cmg.arcpublishing.com/alc/arc-products/pagebuilder/user-docs/pagebuilder-editor-30-resolvers/